algorithms/searching.md

```diff
@@ -66,12 +60,19 @@ E.g. if the array is [4, 1, 6, 8] and n=3, your function should return 4.
 
 ### Challenge
 
-- Write binary search to find a given item in an array. You can write it yourself, or use this tutorial for some guidance: https://www.khanacademy.org/computing/computer-science/algorithms/binary-search/p/challenge-binary-search
-
+- Write a binary search to find a given item in an array. You can write it yourself, or use this tutorial for some guidance: https://www.khanacademy.org/computing/computer-science/algorithms/binary-search/p/challenge-binary-search
 
 ### Check for Understanding
 
 Make a cheat sheet answering the following questions for linear search and binary search:
 - What is it?
 - What types of arrays can you use it on?
 - What is the runtime?
+
+### Supplemental Resources
```

Comment:
Thanks, this is much better 😊

Reply:
ok, I will change all the requested changes.